core/embed/bootloader/CHANGELOG.md

```diff
@@ -4,6 +4,12 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
+## 2.1.9 [September 2024]
+
+### Changed
+
+- [T3T1] Allow starting bootloader without touch controller.  [#4195]
```

Comment:
Not entirely sure about this, its only allowed in manufacturing mode, it shouldn't affect regular users.

Reply:
hmmm ok so how about something like this?

> Ignore touch errors while in manufacturing.

Reply:
i think its better

Reply:
force-pushed